tests/contrib/openai/test_openai_v1.py

```diff
@@ -1346,3 +1346,131 @@ async def handle_async_request(self, request: httpx.Request) -> httpx.Response:
+        model = "gpt-4.1"
+        input_messages = multi_message_input
+        client = openai.OpenAI()
+        client.responses.create(model=model, input=input_messages, tools=[{"type": "web_search_preview"}])
+
+
+@pytest.mark.skipif(
+    parse_version(openai_module.version.VERSION) < (1, 66), reason="Response options only available openai >= 1.66"
+)
+@pytest.mark.snapshot(
+    token="tests.contrib.openai.test_openai.test_response_error",
+    ignores=["meta.http.useragent", "meta.openai.api_type", "meta.openai.api_base"],
+)
+def test_response_error(openai, openai_vcr, snapshot_tracer):
+    """Assert errors when an invalid model is used."""
+    with openai_vcr.use_cassette("response_create_error.yaml"):
+        client = openai.OpenAI()
+        with pytest.raises(openai.BadRequestError):
+            client.responses.create(
+                model="invalid-model",  # Using an invalid model to trigger error
+                input="Hello world",
+                user="ddtrace-test",
+            )
+
+
+@pytest.mark.skipif(
+    parse_version(openai_module.version.VERSION) < (1, 66), reason="Response options only available openai >= 1.66"
+)
+@pytest.mark.snapshot(
+    token="tests.contrib.openai.test_openai.test_response",
+    ignores=["meta.http.useragent", "meta.openai.api_type", "meta.openai.api_base"],
+)
+async def test_aresponse(openai, openai_vcr, snapshot_tracer):
+    """Assert spans are created with async client."""
+    with openai_vcr.use_cassette("response.yaml"):
+        client = openai.AsyncOpenAI()
+        input_messages = multi_message_input
+        await client.responses.create(
+            model="gpt-4.1",
+            input=input_messages,
+            top_p=0.9,
+            max_output_tokens=100,
+            user="ddtrace-test",
+        )
+
+
+@pytest.mark.skipif(
+    parse_version(openai_module.version.VERSION) < (1, 66), reason="Response options only available openai >= 1.66"
+)
+@pytest.mark.snapshot(
+    token="tests.contrib.openai.test_openai.test_response_stream",
+    ignores=["meta.http.useragent", "meta.openai.api_type", "meta.openai.api_base"],
+)
+def test_response_stream(openai, openai_vcr, snapshot_tracer):
+    with openai_vcr.use_cassette("response_stream.yaml"):
+        model = "gpt-4.1"
+        client = openai.OpenAI()
+        resp = client.responses.create(
+            model=model,
+            input="Hello world",
+            stream=True,
+        )
+        _ = [c for c in resp]
+
+
+@pytest.mark.skipif(
+    parse_version(openai_module.version.VERSION) < (1, 66), reason="Response options only available openai >= 1.66"
+)
+@pytest.mark.snapshot(
+    token="tests.contrib.openai.test_openai.test_response_tools_stream",
+    ignores=["meta.http.useragent", "meta.openai.api_type", "meta.openai.api_base"],
+)
+def test_response_tools_stream(openai, openai_vcr, snapshot_tracer):
+    with openai_vcr.use_cassette("response_tools_stream.yaml"):
+        model = "gpt-4.1"
```

Comment:
Since we're not using the model variable again let's just add it directly to the request